content/reference/response-map.adoc

```diff
@@ -1,12 +1,31 @@
 = Response Map
-TBD
-2016-08-22
+Michael Glaesemann
+2016-9-03
 :jbake-type: page
 :toc: macro
 :icons: font
 :section: reference
 
+== Description
```

Comment:
It would be good to indicate that Pedestal may add other keys to this map for its own use, but that applications should treat any such keys and their values as implementation details and their behavior will remain unspecified.